pkg/sql/parse.go

```diff
@@ -66,17 +69,24 @@ func parse(dbms, sqlProgram string) ([]statementParseResult, error) {
 		return sqls, nil
 	}
 
+	left := sqlProgram[:i]
```

Comment:
```go
sqlProgram := sqlProgram[:i]
extendedSQLProgram = sqlProgram[i:]
```

Reply:
Hi @typhoonzero, I am assuming you are suggesting a better naming, which I don't agree.

Please be aware that the `sqlProgram` may contain multiple statements. For example, let's consider the following scenario.

```go
sqlProgram := "select ... to train ...; select ... to train ...;"

sqlProgram := sqlProgram[:i]         // "select ...", which is not a SQL program
extendedSQLProgram = sqlProgram[i:]  // "to train ...; select ... to train", which is not a extendedSQLProgram
```

Reply:
I see. I'm approving this.